src/js/shared_controls.js

```diff
@@ -1059,11 +1065,22 @@ function getMoveDetails(moveInfo, species, ability, item, useMax) {
 		basePower: +moveInfo.find(".move-bp").val(),
 		type: moveInfo.find(".move-type").val()
 	};
+	if (moveName === 'Tera Blast') {
+		// custom logic for stellar type tera blast
+		var isStellar = opts.teraType === 'Stellar';
+		var statDrops = moveInfo.find('.stat-drops');
+		var dropsStats = statDrops.is(':visible');
+		if (isStellar !== dropsStats) {
+			// update stat drop dropdown here
+			if (isStellar) statDrops.show(); else statDrops.hide();
```

Comment:
What does this do? This results in an error, since you cut out statDrops entirely, plus I don't understand why you converted an if statement to a tertiary operator.

Reply:
I reverted the implementation, since doing this violates the linter, and it also messed with some of the formatting.